src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -46,7 +55,12 @@ public static Connection GetDockerConnection(string name)
                 // If SSH connection dialog was cancelled, we should cancel this connection.
                 if (remoteConnection == null)
                     return null;
-                
+
+                // Verify container exists on remote machine.
```

Comment:
Did you want to check in these comments?

Reply:
yes. I'll add TODO in front of it